code/modules/food_and_drinks/restaurant/generic_venues.dm

```diff
@@ -22,8 +22,13 @@
 	var/obj/item/object_to_order = customer_data.get_order(src)
 
 	. = object_to_order
-
-	customer_pawn.say(order_food_line(object_to_order))
+	var/line_to_say = order_food_line(object_to_order)
+	if(prob(1) || SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+		line_to_say = "Yeah can I get a mothafuckin uh"
+		for(var/extra_cheese in 1 to rand(1,69))
+			line_to_say += "h"
+		line_to_say += ", [initial(object_to_order.name)], extra cheese?"
```

Comment:
note that this could confuse people if the ordered thing physically can't have cheese added to it

Reply:
good

Reply:
I mean, drinks with extra cheese are nice I guess?